doc/api/net.markdown

```diff
@@ -51,7 +51,8 @@ var server = net.createServer((socket) => {
 });
 
 // grab a random port.
-server.listen(() => {
+server.listen((err) => {
+  if(err) throw err;
```

Comment:
My personal preference is for a separate line, but do whatever is currently more consistent. I think the docs working group is planning to add linting for this kind of thing in the future.